composer.json

```diff
@@ -11,9 +11,10 @@
     ],
     "require": {
         "php": "^8.1.0",
-        "guzzlehttp/guzzle": "^7.5.0"
+        "psr/http-client-implementation": "1.0"
```

Comment:
I propose to require `psr/http-client-implementation` instead of a specific implementation. So that it is not necessary to exclude guzzle from projects that want to use an other implementation.

For the Laravel package, guzzle can be explicitly required to keep a smooth installation path.

Reply:
this should also require `"php-http/discovery": "^1.15"` so that such an implementation is installed automatically (+ the plugin should be disabled as such since we don't need it in for testing this:
```
    "config": {
        "allow-plugins": {
            "php-http/discovery": false
        }
    },
```

Reply:
If guzzle is required by the laravel package, can we skip the composer plugin?

Reply:
then the plugin will do nothing :shrug: if you fear composer asking for enabling the plugin, it's already whitelisted by both laravel and symfony, and it's anyway still better than composer failing to install because of a missing implementation package.
But you might prefer going with `^1.14` so that this doesn't conflict with sentry when it's installed.